docs/customization/controller.rst

```diff
@@ -122,25 +122,26 @@ getting a list of recommended products from your external api.
             arguments: ['@sylius.repository.product']
             public: true
 
+**4.** Disable autowire for your controller in ``config/services.yaml``
+
+.. code-block:: yaml
+
+    App\Controller\ProductController:
+        autowire: false
+
 .. tip::
 
-    Run ``$ php bin/console debug:container sylius.repository.product`` to check if the class has changed to your implementation.
+    Run ``$ php bin/console debug:container sylius.controller.product`` to check if the class has changed to your implementation.
 
 **4.** Finally you’ll need to add routes in the ``config/routes.yaml``.
 
 .. code-block:: yaml
 
-    app_shop_custom_index:
-        path: /custom/index
-        methods: [GET]
-        defaults:
-            _controller: app.controller.shop.homepage:indexAction
-
-    app_shop_custom_custom:
-        path: /custom/custom
+    app_product_show_index:
+        path: /product/show
         methods: [GET]
         defaults:
-            _controller: app.controller.shop.homepage:customAction
+            _controller: app.controller.product:showAction
```

Comment:
It shouldn't work, this service should still be named `sylius.controller.product`. Current documentation seems ok for me. Can you explain your problem briefly?

Reply:
The documentation is in my opinion a bit misleading. 
In the chapter "How to customize a Resource Controller?" there's described how to create the App\Controller\ProductController to overwrite sylius.controller.product. So it should be mentioned in this chapter how to configure the route for it.

But I understand now, that this doc is heavily relying on the linked pull request und with that it makes more sense, and will think again how to possible improve the docs.  Because in my option there are too much dependencies to the pull request, which does not ONLY override a controller, but also the repository.
So at the moment I have the idea to add a second, simpler pull request to link in the docs which just overrides controller...

Reply:
I would love to review and merge it :) 

And I'm aware that our docs require a few improvements here and there. Thanks for understanding and help :)